beta-pipelines/multiome/atac.wdl

```diff
@@ -0,0 +1,226 @@
+version 1.0
+
+workflow ATAC {
+  meta {
+    description: "Processing for single-cell ATAC-seq data from the level of raw fastq reads. This is the first step of the multiome pipeline. ATAC-seq (Assay for Transposase-Accessible Chromatin using sequencing) is a technique used in molecular biology to assess genome-wide chromatin accessibility. This pipeline accepts fastq files where the cell barcode has been added to the fastq read names as the first field."
+    allowNestedInputs: true
+  }
+
+  input {
+    # Fastq inputs
+    File fastq_gzipped_input_read1
+    File fastq_gzipped_input_read2
+
+    # Trimming options
+    Int min_length
+    Int quality_cutoff
+    Int trim_mem_size = 4
```

Comment:
I recommend including these in the task outputs, but excluding them here. It makes the code cleaner and reduces the inputs. Since you have included `allowNestedInputs: true` you can still set these at the task level without having to specify them here.

Reply:
Makes sense. I made the recommended changes!